Source/Core/VulkanConstants.js

```diff
@@ -0,0 +1,287 @@
+/**
+ * Enum containing Vulkan Constant values by name.
+ * for use when needing VK Formats
```

Comment:
I see that the wording here is modeled after `WebGLConstants.js`, but it's fine to drop that second sentence.

Reply:
Done